plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -567,6 +567,8 @@ private Subject subjectForImageId(String imageId) {
     return null;
   }
 
+  private ThreadLocal<Boolean> retryingAfterCleanup = ThreadLocal.withInitial(()->false);
```

Comment:
maybe, it would make sense to have this setting defined in `che.properties`, so that it would be possible to chnage this behaviour on osio by simply updating cm ?

Reply:
why would we need to change the behaviour ?

Reply:
I do not insist, if was more like thinking outloud ;-) If does not make sense let's keep it as non-tunable option